cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -176,7 +151,7 @@ if [ -f /opt/parallelcluster/.bootstrapped ]; then
     error_exit "This AMI was created with ${!installed_version}, but is trying to be used with ${!cookbook_version}. Please either use an AMI created with ${!cookbook_version} or change your ParallelCluster to ${!installed_version}"
   fi
 else
-  bootstrap_instance
+  error_exit "This AMI was not baked by ParallelCluster. Please use pcluster creatami command to create an AMI by providing your AMI as --parent-image."
```

Comment:
typo: creatami
Then I'd avoid putting cli parameters here, it's hard to rememeber to update them here. We can instead link to createami guide or just saying "providing your AMI as parent image"